namespaces.yaml

```diff
@@ -13,3 +13,15 @@ metadata:
   name: staging
 
 ---
+
+apiVersion: settings.k8s.io/v1alpha1
+kind: PodPreset
+metadata:
+  name: global-proxy-config
+spec:
+  selector:
+    matchLabels:
+      role: proxy
+  envFrom:
+    - configMapRef:
+        name: proxy-config
```

Comment:
This PodPreset won't apply to the relevant namespaces on our cluster. It only applies to the default namespace.
In this file, this block would have to be put in twice with a defining namespace in the metadata:
``` yaml
metadata:
  name: ...
  namespace: production
```

``` yaml
metadata:
  name: ...
  namespace: staging
```